internal/pkg/manifest/errors.go

```diff
@@ -98,6 +98,42 @@ func (e *errFieldMutualExclusive) Error() string {
 	return fmt.Sprintf(`must specify one, not both, of "%s" and "%s"`, e.firstField, e.secondField)
 }
 
+type errGracePeriodSpecifiedInAdditionalRule struct {
+	index int
+}
+type errGracePeriodSpecifiedInAdditionalListener struct {
+	index int
+}
+
+type errGracePeriodsInBothALBAndNLB struct {
+	errFieldMutualExclusive
+}
+
+func (e *errGracePeriodsInBothALBAndNLB) RecommendedActions() string {
+	return fmt.Sprintf(`It looks like you have specified grace_period under both %s and %s. 
+ECS allows "HealthCheckGracePeriodSeconds" as a service level setting and should be specified only once either in http or nlb configuration.`, e.firstField, e.secondField)
```

Comment:
nit: what do you think of the following ⬇️ 
```suggestion
	return fmt.Sprintf(`"grace_period" is a configuration shared by "http" and "nlb". Specify it only once under either "http" or "nlb".`, e.firstField, e.secondField)
```

> It looks like you have specified grace_period under both %s and %s.

I feel like this is an error description instead of a recommended action, and it is `Error()`'s duty to describe an error. I know we do this for some of the other errors, but in this case I think we don't need to be super verbose.

> ECS allows "HealthCheckGracePeriodSeconds" as a service level setting

I'm not sure if this necessarily make sense for users. Seeing this, users may think to themselves - "why is GracePeriods being a service-level setting a reason why you can specify it only once?" 


To understand this causation, the users have to understand how ECS services are hooked with load balancers (i.e. via target group). I feel like this is too deep of an understanding that we want to assume.

Reply:
Yeah I agree with your thought. I think customer doesn't need to think so deep. We can just tell them what correction they need to do. 👍